packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,80 @@
+# Building the app
```

Comment:
we're going to need some more docs on the readme, with instructions like how to setup Appium, etc.

Reply:
it is in README-addTestAndExecute